_drafts/release-workflows.md

```diff
@@ -0,0 +1,309 @@
+- last release from `master` is `1.1.0`
+- last release from `next` is `2.0.0`
+
+Committing a breaking change to `master` would trigger the release of `2.0.0` that would fail as the release already exists.
+
+In addition when the `master` and `next` have the same commit head (after a merging `next` into `master` for example), the first commit done on `next` must be a breaking change.
+Otherwise that would result in a "fork" situation.
+
+For example with the config `"branches": ["master", "next"]` and:
+- last release from `master` is `2.0.0`
+- last release from `next` is `2.0.0`
+
+A `feat` commit on `master` will release version `2.1.0`.
+Committing a `feat` commit on `next` would trigger the release of `2.1.0` that would fail as the release already exists.
+
+With specific channel names:
+
+```json
+{
+  "release": {
+    "branches": [
+      "master",
+      {"branch": "next", "channel": "preview"}
+    ]
+  }
+}
+```
+
+#### LTS branches
+
+Workflow to release legacy versions.
+
+Ideal for modules maintaining legacy versions and doing bug fixes and features only releases.
+
+Branch: `1.x`, `2.x`, `master` (any number of lts branches are supported)
+
+Channel: `1.x`, `2.x`, `latest` if supported by the release target
+
+##### LTS branches workflow
+
+| Action                                          | Result                                                                                                                  |
+|-------------------------------------------------|-------------------------------------------------------------------------------------------------------------------------|
+| **push** to `master`                            | Release on default channel (if supported, otherwise no channel)                                                         |
+| **merge** feature branch to `master`            | Release on default channel (if supported, otherwise no channel)                                                         |
+| **push** to `1.x` or `2.x` branches             | Release on `1.x` or `2.x` channel (if supported, otherwise no channel) if in range, report an error and error otherwise |
+| **merge** any branch to `1.x` or `2.x` branches | Release on `1.x` or `2.x` channel (if supported, otherwise no channel) if in range, report an error and error otherwise |
+
+##### LTS branches configuration
+
+```json
+{
+  "release": {
+    "branches": ["1.x", "2.x", "master"]
+  }
+}
+```
+
+With specific channel names:
+
+```json
+{
+  "release": {
+    "branches": [
+      {"branch": "v1", "range": "1.x", "channel": "v1"},
+      {"branch": "v2", "range": "2.x", "channel": "v2"},
+      "master"
+    ]
+  }
+}
+```
+
+#### Pre-releases
+
+Workflow to distribute releases without incrementing the semantic version during the development of new a application version.  
+Ideal for applications distributing unstable/alpha/preview releases.
+
+Branch: `master`, `beta`, `alpha`
+
+Channel: `latest` if supported by the release target
```

Comment:
I'm not sure I understand this line. Does this imply that `latest` dist-tag will point to the pre-release version?